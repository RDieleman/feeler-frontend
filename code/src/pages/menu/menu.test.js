import React from "react";
import {shallow} from "enzyme";
import MenuPage from "./menu.page";

it("Expect to render Menu page", () => {

    expect(shallow(
        <MenuPage

        />)).toMatchSnapshot();
});