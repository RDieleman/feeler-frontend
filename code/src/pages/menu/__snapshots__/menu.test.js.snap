// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Expect to render Menu page 1`] = `ShallowWrapper {}`;
